index.html

```diff
@@ -288,6 +288,24 @@ <h2>About me</h2>
             </li>
           </ul>
         </div>
+        <section class="contact-section">
+          <div class="contact-info-wrapper">
+              <div class="contact-info">
+                  <h4>Contact Me</h4>
+                  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+              </div>
+              <div class="contact-info">
+                <form id="form" action="https://formspree.io/f/mwkakrqj" method="POST">
+                  <input class="group" type="text" placeholder="Daniel Samuel" maxlength="30"
+                  required /><br>
+                  <input class="group" type="email" placeholder="dansamexample@gmail.com" maxlength="30"
+                  required /><br>
+                  <input class="text-group" type="text" placeholder="Write your message here"/><br>
```

Comment:
- [ ] It's recommended using `textarea` tag here, instead of the `input` tag, because this input field is supposed to give a user enough room to leave their message, thoughts, or ideas. Please fix it. 😄 
- [ ] Also, remember to give the `textarea` element an attribute of `maxlength="500". This is to limit the number of characters a user can give at one given moment.